docs/debug+profile/clang/addresssanitizer.md

```diff
@@ -0,0 +1,38 @@
+---
+title: AddressSanitizer (Clang Sanitisers)
+---
+
+The [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) is integrated into [Clang](https://clang.llvm.org/) 3.4+ and therefore, if Clang is installed, using it with Mono's `autogen.sh` is as simple as running the following:
+
+``` bash
+$ cd /home/root/of/mono
+$ ./autogen.sh --prefix=/usr/local CC=clang CFLAGS=-fsanitize=address LDFLAGS=-fsanitize=address CXX=clang++
+$ make
+...
+```
+
+Please keep in mind that `CXX` has to be set to `clang++` to avoid unwanted side effects and compiling errors.
+
+After `autogen.sh` has been executed as suggested above, the following `make` command uses Clang and injects necessary reporting functions. Later, `make` uses [roslyn](https://github.com/dotnet/roslyn) to compile `mcs` with the freshly built Mono runtime.
```

Comment:
I removed the last sentence in the AddressSanitizer's article and slightly altered it in the ThreadSanitizer's article (mentioning the `mscorelib.dll` directly instead of the whole `mcs/` folder)